ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -313,3 +318,39 @@ class TabBarItemView: UIView {
         }
     }
 }
+
+// MARK: - TabBarItemView UIPointerInteractionDelegate
+
+extension TabBarItemView: UIPointerInteractionDelegate {
+    @available(iOS 13.4, *)
+    func pointerInteraction(_ interaction: UIPointerInteraction, styleFor region: UIPointerRegion) -> UIPointerStyle? {
+        let pointerEffect = UIPointerEffect.highlight(.init(view: self))
+
+        var pointerFrame = imageView.frame
+
+        if titleLabel.superview != nil {
```

Comment:
is this logic correct in compact mode when title is below the image view?

Reply:
your origin might be fine but not the width and height

Reply:
Yes, the logic is very flexible. It would work no matter where the title and the icon are placed within the TabBarItemView. It works for TabBarView compact/regular with/without titles and in SideTabBar with/without titles.

Reply:
Width and height will work, is there something that you see that makes you think it won't work?
The formula is:
width = max(title.maxX, title.maxX) - min(title.minX, title.minX)
height = max(title.maxY, title.maxY) - min(title.minY, title.minY)